libevmcore/Instruction.cpp

```diff
@@ -164,152 +173,160 @@ const std::map<std::string, Instruction> dev::eth::c_instructions =
+	{ Instruction::SGT,          { "SGT",            0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::EQ,           { "EQ",             0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::ISZERO,       { "ISZERO",         0,     1,    1,  false,       VeryLowTier } },
+	{ Instruction::AND,          { "AND",            0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::OR,           { "OR",             0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::XOR,          { "XOR",            0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::BYTE,         { "BYTE",           0,     2,    1,  false,       VeryLowTier } },
+	{ Instruction::ADDMOD,       { "ADDMOD",         0,     3,    1,  false,       MidTier } },
+	{ Instruction::MULMOD,       { "MULMOD",         0,     3,    1,  false,       MidTier } },
+	{ Instruction::SIGNEXTEND,   { "SIGNEXTEND",     0,     2,    1,  false,       LowTier } },
+	{ Instruction::SHA3,         { "SHA3",           0,     2,    1,  false,       SpecialTier } },
+	{ Instruction::ADDRESS,      { "ADDRESS",        0,     0,    1,  false,       BaseTier } },
+	{ Instruction::BALANCE,      { "BALANCE",        0,     1,    1,  false,       SpecialTier } },
+	{ Instruction::ORIGIN,       { "ORIGIN",         0,     0,    1,  false,       BaseTier } },
+	{ Instruction::CALLER,       { "CALLER",         0,     0,    1,  false,       BaseTier } },
+	{ Instruction::CALLVALUE,    { "CALLVALUE",      0,     0,    1,  false,       BaseTier } },
+	{ Instruction::CALLDATALOAD, { "CALLDATALOAD",   0,     1,    1,  false,       VeryLowTier } },
+	{ Instruction::CALLDATASIZE, { "CALLDATASIZE",   0,     0,    1,  false,       BaseTier } },
+	{ Instruction::CALLDATACOPY, { "CALLDATACOPY",   0,     3,    0,  true,        VeryLowTier } },
+	{ Instruction::CODESIZE,     { "CODESIZE",       0,     0,    1,  false,       BaseTier } },
+	{ Instruction::CODECOPY,     { "CODECOPY",       0,     3,    0,  true,        VeryLowTier } },
+	{ Instruction::GASPRICE,     { "GASPRICE",       0,     0,    1,  false,       BaseTier } },
+	{ Instruction::EXTCODESIZE,  { "EXTCODESIZE",    0,     1,    1,  false,       SpecialTier } },
+	{ Instruction::EXTCODECOPY,  { "EXTCODECOPY",    0,     4,    0,  true,        SpecialTier } },
+	{ Instruction::BLOCKHASH,    { "BLOCKHASH",      0,     1,    1,  false,       ExtTier } },
+	{ Instruction::COINBASE,     { "COINBASE",       0,     0,    1,  false,       BaseTier } },
+	{ Instruction::TIMESTAMP,    { "TIMESTAMP",      0,     0,    1,  false,       BaseTier } },
+	{ Instruction::NUMBER,       { "NUMBER",         0,     0,    1,  false,       BaseTier } },
+	{ Instruction::DIFFICULTY,   { "DIFFICULTY",     0,     0,    1,  false,       BaseTier } },
+	{ Instruction::GASLIMIT,     { "GASLIMIT",       0,     0,    1,  false,       BaseTier } },
+	{ Instruction::JUMPTO,       { "JUMPTO",         0,     1,    0,  true,        VeryLowTier } },
```

Comment:
This should have an "add" value of 4, because it has a 4-byte immediate argument, similar for other opcodes.